packages/monitoring/src/main/components/EndpointComponent.ts

```diff
@@ -60,10 +63,14 @@ export default class EndpointComponent implements HealthComponent {
         .timeout(timeout as number)
         .retry(retries, (err, res) => {
           attemptsCount += 1
-          if (err && this.logger) {
+          if (err) {
             this.logger.info(
               `Attempting to get health of external service '${name}', got: ${err.code} - ${err.message}`,
             )
+          } else if (res?.statusType === STATUS_4XX || res?.statusType === STATUS_5XX) {
```

Comment:
i think this is more clearly written as
```typescript
res.clientError || res.serverError
```
with no need for custom `STATUS_4XX ` constants